processing/sql/dml/009_prg_pa_update.sql

```diff
@@ -0,0 +1,11 @@
+-- aktualizacja simc na podstawie terytu
+-- jezeli nadal simc jest nieokreslone, to wybieramy na zasadzie podobienstwa nazwy miejscowosci
+update prg.pa
+set teryt_simc = (
+	select simc.sym
+	from teryt.simc as simc
+	where left(pa.teryt7, 6) = simc.woj || simc.pow || simc.gmi and similarity(simc.nazwa, pa.msc) >= 0.3
```

Comment:
pa.tery6,
similarity 0.3 to chyba dość niski próg?

Reply:
Wartość 0.3 jest wartkością domyślną funkcji zalecana przez autorów (https://www.postgresql.org/docs/9.1/pgtrgm.html). Weryfikowałem, na rzeczywistych danych i w kilku przypadkach funkcja sie "pomyliła", Było ich jednak zaledwie klika. Oczywiście zawsze można podnieść ten limit. Jestem otwarty na propozycje.